### ISSUE TITLE
Add Aircraft for Mexico: AT-33A Shooting Star

### ISSUE THREAD

**Issue by RevanstheBomb** [PygmalionOfCyprus/cmo-db-requests#51]
### DB Selector

DB3K

### Hypothetical

No

### Name

AT-33A Shooting Star

### Country

Mexico

### Service

Air Force

### In Commission

1961-2012

### Length (m)

11.51

### Wingspan (m)

11.849

### Height (m)

3.56

### Empty Weight (kg)

3794

### Maximum Takeoff Weight (MTOW) (kg)

6832

### Crew

2

### Propulsion

1x Allison J33-A-35

### Service Ceiling (ft)

48,000

### Military/Afterburner Speed (kt)

520

### Cruise Speed (kt)

395kt

### Supercruise

No

### Ferry Range (nm)

1107.944nm

### Sensors

1x Mk1 Eyeball

### Mounts

2x 12.7mm/50 M3 x 6 [300 rnds x 6]

### Loadout Options

-1x 225 USG Drop Tank, 
2x 250 ImpG Wingtip Tank
(Ferry)

-14x HYDRA 70mm Rockets,
12.7mm/50 MG
(Strike, Ground/Naval)

-8x Mk81 LDGP
(Strike, Ground/Naval)

-4x Mk82 LDGP
(Strike, Ground/Naval)

### Comms/Datalinks

UHF/VHF Radio [Unsecure]

### Aircraft Codes

Bombsight - Basic

### Sources

https://es.wikipedia.org/wiki/Fuerza_A%C3%A9rea_Mexicana

http://www.cactusairforce.com/inventory_item/t-33a-shooting-star/#tab-id-3

https://www.marines.mil/Portals/1/Publications/Mexico%20Study_4.pdf

https://en.wikipedia.org/wiki/Lockheed_T-33

**Comment by PygmalionOfCyprus** [comment]
Added, mostly.

I've swapped out the gun for 2x M3s firing 350rnds/g since that's what all the sources say.

Also, as far as I can tell, the AT-33A only had two hardpoints -- so I'm a bit confused by the 8x/4x bomb loadouts (obviously bombs, especially the Mk81, can be multi-loaded onto a single hardpoint, but I don't know that that occurred with the T-33...). This is supported by your sources, which all mention only 2 hardpoints and 2 bombs. Likewise, I don't see any record of the 225 USG drop tank, just the iconic wingtips.

Anyway, I've added a set of loadouts for 2x hardpoints: 14x HYDRA rockets; 2x Mk81s; 2x Mk82s. Also, a second set with the wingtip tangs for longer-range operations.

Feel free to comment if I missed something; loadouts are easy to tweak. Otherwise, I'm gonna call this one closed.
